app/packs/controllers/no_newlines_controller.js

```diff
@@ -0,0 +1,9 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
```

Comment:
How about adding a comment about what this controller does.  Then feel free to merge.